.travis.yml

```diff
@@ -186,8 +186,11 @@ deploy:
     # Both the build and deploy steps for Emscripten are only run within the Ubuntu
     # configurations (not for macOS).  That is controlled by conditionals within the bash
     # scripts because TravisCI doesn't provide much in the way of conditional logic.
+    # This is also the deploy target for the dockerfile. If we are pushing into a develop branch, it will be tagged
+    # as a nightly and appended the commit of the branch it was pushed in. If we are pushing to master it will
+    # be tagged as "stable" and given the version tag as well.
     - provider: script
-      script: test $TRAVIS_EMSCRIPTEN != On || scripts/release_emscripten.sh
+      script: test $SOLC_EMSCRIPTEN == On && scripts/release_emscripten.sh || test $SOLC_DOCKER != On || ./scripts/docker_deploy.sh
```

Comment:
I don't think it will work that way. If the emscripten release succeeded, it will not do the docker release.